index.js

```diff
@@ -49,67 +50,68 @@ client.on("message", function (message) {
     inventoryCommand(message);
   } else if (message.content.startsWith("!lookup")) {
     lookUpCommand(message);
+  } else if (message.content.startsWith("!remove")) {
+    removeCardCommand(message);
   }
 });
 
 const drawCommand = (message) => {
-  var ref = db.ref("users");
+  const ref = db.ref("users");
   ref.once("value", function (snapshot) {
     const userData = snapshot.val();
+    let userID;
+    let userInventory;
```

Comment:
this is one solution, but we can avoid having to use `let` by just moving this further down,